compoments/report-repair/summary.js

```diff
@@ -0,0 +1,58 @@
+import React, {useEffect, useState} from 'react';
+import SummaryList from '../summaryList';
+import Button from '../button';
+const Summary = ({values, changeLinkUrlValues}) => {
+  let [whatIsRepairProblemLink, setWhatIsRepairProblemLink] = useState(null)
+  let [describeRepairProblemLink, setDescribeRepairProblemLink] = useState(null)
+
+  const convertDateToDisplayDate = (date) => {
+    let dateArray = date?.split(' ')
+    dateArray?.splice(3, 0, 'between')
+    return(dateArray?.join(' '))
+  }
+
+  useEffect(()=>{
+    setWhatIsRepairProblemLink(changeLinkUrlValues[values.repairLocation])
+    setDescribeRepairProblemLink(changeLinkUrlValues[values.repairProblem])
+    convertDateToDisplayDate(values.availability)
+  })
+  const personalDetailsSummary = [
+    {pageName:'Repair address', value: values.address?.display, link: 'postcode'},
+    {pageName:'Appointment contact number', value: values.contactPersonNumber, link: 'contact-person'},
+  ]
+  const repairDetailsSummary = [
+    { pageName:'Where is the problem?', value: values.repairLocation, link:'repair-location'},
+    { pageName:'What is the problem?', value: values.repairProblem, link: changeLinkUrlValues[values.repairLocation]},
+    { pageName:'What best describes the problem?', values: values.repairProblemBestDescription, link: changeLinkUrlValues[values.repairProblem]},
+    { pageName:'Description', value: values.description?.text, link:'repair-description'}
+  ]
+  const appointmentDetailsSummary = [
+    { pageName:'Date', value: convertDateToDisplayDate(values.availability), link:'repair-availability'},
+    { pageName:'Confirmation contact details', value: values.contactDetails?.input, link: 'contact-details'},
+  ]
+
+  return(
+    <>{values && whatIsRepairProblemLink && describeRepairProblemLink && (
+      <div>
+        <div className="govuk-grid-row govuk-body-m">
+          <div className="govuk-grid-column-two-thirds">
+            <h1 className='govuk-heading-xl'>Request summary</h1>
+
+            <h2 className="govuk-heading-m">Personal details</h2>
+            <SummaryList summary={personalDetailsSummary}/>
+
+            <h2 className="govuk-heading-m">Repair details</h2>
+            <SummaryList summary={repairDetailsSummary}/>
+
+            <h2 className="govuk-heading-m">Appointment details</h2>
+            <SummaryList summary={appointmentDetailsSummary}/>
+
+          </div>
+        </div>
+        <Button>continue</Button>
```

Comment:
This should start with a capital letter.
```suggestion
        <Button>Continue</Button>
```